### PR TITLE
Update CSV

### PR DESCRIPTION
**Pull request by g2vinay**

### CODE REVIEW

File: _data/releases/latest/dotnet-packages.csv [hunk 1]
```diff
@@ -8,7 +8,7 @@
 "Azure.MixedReality.ObjectAnchors.Conversion","","0.1.0-beta.1","Azure Object Anchors Conversion","Mixed Reality","objectanchors","","","client","true","","","",""
 "Azure.MixedReality.RemoteRendering","1.0.0","","Azure Remote Rendering","Mixed Reality","remoterendering","","","client","true","","","",""
 "Azure.Security.Attestation","","1.0.0-beta.1","Azure Security Attestation","Microsoft Azure Attestation","attestation","","","client","True","","","",""
-"Azure.Monitor.OpenTelemetry.Exporter","","1.0.0-beta.2","Azure.Monitor.OpenTelemetry.Exporter","monitor","monitor","","","client","true","","","",""
+"Azure.Monitor.OpenTelemetry.Exporter","","1.0.0-beta.2","Azure.Monitor.OpenTelemetry.Exporter","Monitor OpenTelemetry Exporter","monitor","","","client","true","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:11]:
```suggestion
"Azure.Monitor.OpenTelemetry.Exporter","","1.0.0-beta.2","Monitor OpenTelemetry Exporter","Monitor","monitor","","","client","true","","","",""
```

File: _data/releases/latest/dotnet-packages.csv [hunk 2]
```diff
@@ -18,7 +18,7 @@
 "Azure.Core","1.10.0","","Core","Core","core","","","client","true","","10/29/2019","",""
 "Azure.Core.Amqp","1.0.0","","Core - AMQP","Core","core","","","client","true","","11/23/2020","",""
 "Azure.Cosmos","","4.0.0-preview3","Cosmos DB","Cosmos DB","https://github.com/Azure/azure-cosmos-dotnet-v3/tree/releases/4.0.0-preview3","https://docs.microsoft.com/dotnet/api/azure.cosmos","","client","true","","","","Will Replace: Microsoft.Azure.Cosmos"
-"Azure.DigitalTwins.Core","1.2.1","","Digital Twins - Core","Digital Twins","digitaltwins","","","client","true","","10/30/2020","",""
+"Azure.DigitalTwins.Core","1.2.1","","Digital Twins - Core","Digital Twins","IoT","","","client","true","","10/30/2020","",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:21]:
```suggestion
"Azure.DigitalTwins.Core","1.2.1","","Digital Twins - Core","IoT","digitaltwins","","","client","true","","10/30/2020","",""
```

**Comment by weshaggard** [comment]
@g2vinay it looks like you might have edited the wrong column for a number of these.

File: _data/releases/latest/java-packages.csv [hunk 3]
```diff
@@ -19,7 +19,7 @@
 "azure-core-serializer-json-gson","com.azure","1.1.0","","Core Serializer GSON JSON","Core","core","","","client","true","","",""
 "azure-core-serializer-json-jackson","com.azure","1.2.0","","Core Serializer Jackson JSON","Core","core","","","client","true","","",""
 "azure-cosmos","com.azure","4.12.0","","Cosmos DB","Cosmos DB","cosmos","","","client","true","","","Replaces: azure-cosmos 3.x"
-"azure-digitaltwins-core","com.azure","1.0.3","","Digital Twins - Core","Digital Twins","digitaltwins","","","client","true","","",""
+"azure-digitaltwins-core","com.azure","1.0.3","","Digital Twins - Core","Digital Twins","IoT","","","client","true","","",""
```
Comment (weshaggard) [inline _data/releases/latest/java-packages.csv:22]: You changed the repo path not the servicename

File: _data/releases/latest/java-packages.csv [hunk 4]
```diff
@@ -33,8 +33,8 @@
...
 "azure-ai-metricsadvisor","com.azure","","1.0.0-beta.3","Metrics Advisor","Metrics Advisor","metricsadvisor","","","client","true","1.0.0b4,04/06/2021","",""
 "azure-security-attestation","com.azure","","1.0.0-beta.1","Microsoft Azure SDK for Attestation","Microsoft Azure Attestation","attestation","","","client","true","","",""
-"azure-monitor-opentelemetry-exporter","com.azure","","1.0.0-beta.3","Monitor Exporter for OpenTelemetry","Monitor","monitor","","","client","true","1.0.0,04/13/2021","",""
-"opentelemetry-exporters-azuremonitor","com.azure","","1.0.0-beta.1","Monitor Exporter for OpenTelemetry","Monitor","monitor","","","client","true","","",""
+"azure-monitor-opentelemetry-exporter","com.azure","","1.0.0-beta.3","Monitor Exporter for OpenTelemetry","Monitor OpenTelemetry Exporter","monitor","","","client","true","1.0.0,04/13/2021","",""
+"opentelemetry-exporters-azuremonitor","com.azure","","1.0.0-beta.1","Monitor Exporter for OpenTelemetry","Monitor OpenTelemetry Exporter","monitor","","","client","true","","",""
```
Comment (weshaggard) [inline _data/releases/latest/java-packages.csv:37]: I think we wanted to change the display name not the service name in this case.

File: _data/releases/latest/dotnet-packages.csv [hunk 5]
```diff
@@ -8,7 +8,7 @@
 "Azure.MixedReality.ObjectAnchors.Conversion","","0.1.0-beta.1","Azure Object Anchors Conversion","Mixed Reality","objectanchors","","","client","true","","","","",""
 "Azure.MixedReality.RemoteRendering","1.0.0","","Azure Remote Rendering","Mixed Reality","remoterendering","","","client","true","","03/02/2021","","",""
 "Azure.Security.Attestation","","1.0.0-beta.1","Azure Security Attestation","Microsoft Azure Attestation","attestation","","","client","True","","","","",""
-"Azure.Monitor.OpenTelemetry.Exporter","","1.0.0-beta.2","Azure.Monitor.OpenTelemetry.Exporter","monitor","monitor","","","client","true","","","","",""
+"Azure.Monitor.OpenTelemetry.Exporter","","1.0.0-beta.2","Monitor OpenTelemetry Exporter","monitor","monitor","","","client","true","","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/dotnet-packages.csv:11]:
```suggestion
"Azure.Monitor.OpenTelemetry.Exporter","","1.0.0-beta.2","Monitor OpenTelemetry Exporter","Monitor","monitor","","","client","true","","","","",""
```

**Comment by g2vinay** [comment]
@weshaggard  yep, I realized the column mismatch after opening the PR.
Fixed it.

File: _data/releases/latest/java-packages.csv [hunk 6]
```diff
@@ -33,8 +33,8 @@
...
 "azure-ai-metricsadvisor","com.azure","","1.0.0-beta.3","Metrics Advisor","Metrics Advisor","metricsadvisor","","","client","true","1.0.0b4,04/06/2021","","","",""
 "azure-security-attestation","com.azure","","1.0.0-beta.1","Microsoft Azure SDK for Attestation","Microsoft Azure Attestation","attestation","","","client","true","","","","",""
-"azure-monitor-opentelemetry-exporter","com.azure","","1.0.0-beta.4","Monitor Exporter for OpenTelemetry","Monitor","monitor","","","client","true","1.0.0,04/13/2021","","","",""
-"opentelemetry-exporters-azuremonitor","com.azure","","1.0.0-beta.1","Monitor Exporter for OpenTelemetry","Monitor","monitor","","","client","true","","","","",""
+"azure-monitor-opentelemetry-exporter","com.azure","","1.0.0-beta.4","Monitor OpenTelemetry Exporter","Monitor","monitor","","","client","true","1.0.0,04/13/2021","","","",""
+"opentelemetry-exporters-azuremonitor","com.azure","","1.0.0-beta.1","Monitor OpenTelemetry Exporter","Monitor","monitor","","","client","true","","","","",""
```
Comment (weshaggard) [inline _data/releases/latest/java-packages.csv:37]: I don't sure who to ask but I suspect this is an older name and thus we should just hide it by setting "true" in the Hide column.

**Review by weshaggard — APPROVED** [review]
1 suggestion and one question but otherwise looks good.

**Pull request merged**